app/furniture/marketplace/policy.rb

```diff
@@ -2,7 +2,7 @@ class Marketplace
   class Policy < ApplicationPolicy
     def create?
       return true if current_person.operator?
-      return true if current_person.member_of?(marketplace.space)
+      return true if current_person.member_of?(space)
```

Comment:
This allows us to have `Models` within the `Marketplace` that do not reference a `Marketplace` directly, such as `TaxRate` and `Tag`